src/cljs/org/broadinstitute/firecloud_ui/page/workspace/details.cljs

```diff
@@ -7,6 +7,7 @@
     [org.broadinstitute.firecloud-ui.page.workspace.workspace-data :refer [render-workspace-data]]
     [org.broadinstitute.firecloud-ui.page.workspace.method-configs :refer [render-method-configs]]
     [org.broadinstitute.firecloud-ui.paths :as paths]
+    [org.broadinstitute.firecloud-ui.page.workspace.submissions :as submissions]
```

Comment:
I had put "s" (for "submissions") between "paths" and "utils"....but does it belong between "method-configs" and "paths"?